app-frontend/src/app/components/navBar/navBar.controller.js

```diff
@@ -30,23 +34,4 @@ export default class NavBarController {
     logout() {
         this.authService.logout();
     }
-
-    selectProjectModal() {
```

Comment:
On top of Lucien's comment... The quick switch modal was limited in what we could display. Modals in general are very difficult to get right and I don't feel like our quick switch modal is amazing. It also feels more more consistent for the workflow if opening projects all happened front the same location.